services/galley/src/Galley/Cassandra/TeamFeatures.hs

```diff
@@ -27,8 +27,9 @@ import Cassandra
 import qualified Cassandra as C
 import Control.Monad.Trans.Maybe
 import Data.Id
+import Data.Misc (HttpsUrl)
 import Data.Proxy
-import Data.Time (UTCTime)
+import Data.Time (NominalDiffTime)
```

Comment:
What are the semantics of NominalDiffTime?

Reply:
It's a Haskell type the represents a length of time: https://hackage.haskell.org/package/time-1.12.2/docs/Data-Time-Clock.html#t:NominalDiffTime.

This is just the internal type. In the config and in the API this is just an integer representing seconds.